### PR TITLE
version sub-command modified

### PR DESCRIPTION
**Pull request by AnirudhPokala**
The version sub-command is printing the current version of CLI being used.

### CODE REVIEW

File: cmd/version.go [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Current version of CLI being used",
+	Long:  "Gives the current PF9CTL version",
 	Run: func(cmd *cobra.Command, args []string) {
-		zap.S().Info("Version called")
+		zap.S().Debug("Version called")
+		fmt.Println("PF9CTL v1.0.0")
```
Comment (anupbarve) [inline cmd/version.go:19]: Put the version string in constants.go.

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: cmd/version.go (hunk 1 above)
Comment (sahil-lakhwani) [inline cmd/version.go:19]: should the CLI be always addressed in lower case - `pf9ctl`?

Comment (AnirudhPokala) [reply cmd/version.go]: done anup added the version string in constants.go

Comment (AnirudhPokala) [reply cmd/version.go]: changed sahil to lower case.

**Review by anupbarve — APPROVED** [review]
As a general rule, please add the console output in the description field going forward.
LGTM

**Comment by AnirudhPokala** [comment]
> As a general rule, please add the console output in the description field going forward.
> LGTM

Sure 
[centos@pokala f1]$ sudo ./pf9ctl version
pf9ctl v1.0

**Pull request merged**